index.xml

```diff
@@ -242,6 +275,21 @@ MUST be ignored.
 
    <section anchor="canonicalization" title="Signature String Construction">
 
+   <t>
+A signed HTTP message needs to be tolerant of some trivial
+alterations during transmission
+as it goes through gateways, proxies, and other entities. These changes are often
```

Comment:
@liamdennehy nitpicking: "gateways, proxies and other entities" rfc 7230 probably uses the term Intermediaries https://tools.ietf.org/html/rfc7230#section-2.3

Reply:
I pick nits for a living, more than happy to be on the receiving end. Correcting in a general quality and fixup PR.